src/app/core/session/session-service/synced-session.service.ts

```diff
@@ -89,48 +89,47 @@ export class SyncedSessionService extends SessionService {
         const syncPromise = this.sync(); // no liveSync() here, as we can't know when that's finished if there are no changes.
 
         // no matter the result of the non-live sync(), start liveSync() once it is done
-        syncPromise.then(
-          // successful -> start liveSync()
-          () => this.liveSyncDeferred(),
-          // not successful -> only start a liveSync() to retry, if we are logged in locally
-          // otherwise the UI is in a fairly unusable state.
-          async () => {
+        syncPromise
+          .then(() => this.liveSyncDeferred())    // successful -> start liveSync()
+          .catch(async () => {
+            // not successful -> only start a liveSync() to retry, if we are logged in locally
+            // otherwise the UI is in a fairly unusable state.
             if ((await localLogin) === LoginState.LOGGED_IN) {
               this.liveSyncDeferred();
             } else {
-              // TODO(lh): Alert the AlertService: Your password was changed recently, but there is an issue with sync. Try again later!
+                // TODO(lh): Alert the AlertService: Your password was changed recently, but there is an issue with sync. Try again later!
             }
-          }
-        );
+          });
 
         // asynchronously check if the local login failed --> this happens, when the password was changed at the remote
         localLogin.then(async (loginState: LoginState) => {
           if (loginState === LoginState.LOGIN_FAILED) {
             // in this case: when the sync is completed, retry the local login after the sync
-            await syncPromise;
+            try {
+              await syncPromise;
```

Comment:
At least in some test scenarios this promised rejected what resulted in the later login process not being executed. The login process is now executed no matter what the promise returns. I am not 100% sure if this is the expected behavior.